src/rules/no-default-export.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    // ignore non-modules
+    if (context.parserOptions.sourceType !== 'module') {
+      return {}
+    }
+
+    var message = 'Prefer named exports.'
+
+    return {
+      ExportDefaultDeclaration: function (node) {
+        context.report({node, message})
+      },
+
+      ExportNamedDeclaration: function (node) {
+        for (var i = 0; i < node.specifiers.length; i++) {
+          var specifier = node.specifiers[i]
```

Comment:
this should use `node.specifiers.forEach` instead of a loop (which will allow both `var` instances to be removed)